docs/configuration.rst

```diff
@@ -614,6 +614,8 @@ server-host
   * :code:`*6` - any IPv4 or IPv6 hostname, IPv6 preferred
   * :code:`!6` - any IPv6 hostname
 
+  If you are using multiple network interfaces, you need to specify a unique hostname in the configuration of each PostgREST instance.
```

Comment:
Hm, it's not really true as the instances could work anyway. I think we should leave this part out. The most important thing is to point the multiple network card limitation on the health check.

```suggestion
```